mediachain/datastore/utils.py

```diff
@@ -61,6 +61,9 @@ def bytes_for_object(obj):
 
 def object_for_bytes(obj_bytes):
     try:
-        return cbor.loads(obj_bytes)
+        decoded = cbor.loads(obj_bytes)
+        if isinstance(decoded, dict):
```

Comment:
yeah, this is to avoid the thing where it decodes "hello world" as a cbor unicode string "ello worl" 😄